pandas/tests/series/test_constructors.py

```diff
@@ -303,9 +299,27 @@ def test_constructor_pass_nan_nat(self):
     def test_constructor_cast(self):
```

Comment:
@ucals : You broke up the tests, but your new tests still use `pytest.raises` when `tm.assert_raises_regex` would be more useful.